### DISCUSSION TITLE
Lotus run dozens of process and use 100% CPU. How to fully uninstall it ?

### DISCUSSION THREAD

**Discussion by adwulfran** [filecoin-project/lotus#8269] [Build and Sync] [answered]
### Checklist

- [x] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [x] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [x] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [x] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [x] lotus daemon - chain sync
- [x] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [X] lotus miner - proving(WindowPoSt)
- [X] lotus miner/market - storage deal
- [X] lotus miner/market - retrieval deal
- [X] lotus miner/market - data transfer
- [X] lotus client
- [X] lotus JSON-RPC API
- [X] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
lastest
```

### Describe the Bug

Lotus run dozens of process and use 100% CPU.

### Logging Information

```text
Htop shows lotus runnings dozens of process and using 100% CPU
```

**Comment by Reiers** [comment, marked as answer]
Hi @adwulfran 

Seems like something went wrong somewhere - Im moving this from Lotus issues to Discussions for help and troubleshooting. Since this clearly a user error and not a lotus issue.

If its running in the background you might have started lotus as a service, then the cmd is:
`systemctl stop lotus-daemon`

Regular stop:
`lotus daemon stop
`
`lotus-miner stop`

>  How to fully uninstall it ?
- Quick google search and you will find a way to fully remove it. 
```
sudo apt-get remove application
sudo apt-get remove application*

sudo apt-get remove --purge application
sudo apt-get remove --purge application*

sudo apt-get purge application
sudo apt-get purge application*
```

**Comment by SgtCoin** [comment]
@adwulfran 

I agree with @Reiers here. How are things looking on your end? Did you get lotus removed successfully?

**Comment by adwulfran** [comment]
@SgtCoin 
Yes I removed it successfully, thanks @Reiers.
I had gigabytes of storage in /snap due to lotus that was harder to remove.
